bulbs/videos/migrations/0002_auto_20160613_1116.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
Ah, I guess this new migration is needed after I reverted to the default ID behavior. Sure, let's add it